x-pack/plugins/apm/public/components/app/service_details/service_detail_tabs.tsx

```diff
@@ -23,7 +23,7 @@ import { ServiceMap } from '../ServiceMap';
 import { ServiceMetrics } from '../service_metrics';
 import { ServiceNodeOverview } from '../ServiceNodeOverview';
 import { ServiceOverview } from '../service_overview';
-import { TransactionOverview } from '../TransactionOverview';
+import { TransactionOverview } from '../transaction_overview';
```

Comment:
I'd love if there was a way to automatically rename all our files once and for all.

Reply:
maybe we could create a new issue to tackle all the renaming once and for all...